.github/actions/is-release-in-progress-v1/README.md

```diff
@@ -0,0 +1,23 @@
+# is-release-in-progress-v1
+
+A GitHub Action to determine if a parent GHA should continue to run or stop.
+
+## Inputs
+
+| Name          | Required | Description         |
+| ------------- | -------- | ------------------- |
+| `githubToken` | yes      | GitHub Secret Token |
+
+## Outputs
+
+| Name                  | Description                                         |
+| --------------------- | --------------------------------------------------- |
+| `isReleaseInProgress` | `true` if an open RC or a release PR already exists |
+
+## Example usage
+
+```yaml
+uses: dequelabs/axe-api-team-public/.github/actions/is-release-in-progress-v1@main
+with:
+  githubToken: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I think to stay consistent with other GHA we should follow [kebab casing](https://www.freecodecamp.org/news/snake-case-vs-camel-case-vs-pascal-case-vs-kebab-case-whats-the-difference/#kebab-case) instead of camel case when using the action in yml